virttest/utils_sys.py

```diff
@@ -74,6 +75,33 @@ def get_host_bridge_id(session=None):
     return host_bridges if host_bridges else []
 
 
+def is_image_mode(check_command="rpm-ostree", session=None):
```

Comment:
check_command parameter is not needed because the user don't need to know how to check it